packages/xo-web/src/common/xo/index.js

```diff
@@ -345,9 +345,15 @@ export const subscribeRoles = createSubscription(
 
 export const subscribeIpPools = createSubscription(() => _call('ipPool.getAll'))
 
-export const subscribeResourceCatalog = createSubscription(() =>
-  _call('cloud.getResourceCatalog')
-)
+export const subscribeResourceCatalog = ({ filters } = {}) =>
+  createSubscription(() =>
+    _call('cloud.getResourceCatalog', { filters }).then(catalog => {
+      if (catalog !== undefined) {
+        delete catalog._namespaces
```

Comment:
https://github.com/vatesfr/xen-orchestra/pull/4442#discussion_r327094745

Reply:
This function is used only in `xo-web` and and in `xo-web` don't need it whether in the present or in the future :slightly_smiling_face:

Reply:
You're right, I've followed Badr without thinking of side effects :worried:

Reply:
Yes, comments are suggestions, but when a dev introduces some changes, it's their job to make sure it does not break existing code :slightly_smiling_face:

Reply:
Also, deleting the property as you previously did was also impacting the code because objects are passed by reference.

You may need to create a filtered copy here (maybe by using [`_.omit`](https://devdocs.io/lodash~4/index#omit)).